hardware/edid.ksy

```diff
@@ -257,6 +267,195 @@ types:
         1: ratio_4_3
         2: ratio_5_4
         3: ratio_16_9
+  dtd_timing:
+    seq:
+      - id: pixel_clock_mod
+        type: u2
+        doc: |
+          Pixel Clock / 10,000
```

Comment:
I'd omit this, given that it's well enough described by the dedicated `pixel_clock` instance:

https://github.com/kaitai-io/kaitai_struct_formats/blob/36c7af9da09f62b85d8a9a93cf0d93bb06cca93a/hardware/edid.ksy#L334-L335

Reply:
Fixed in https://github.com/N-Nagorny/kaitai_struct_formats/commit/e1edfef38739c85c8ea636a265dea8784f4e6490